contracts/BaseJumpRateModelV2.sol

```diff
@@ -172,13 +172,12 @@ abstract contract BaseJumpRateModelV2 is InterestRateModel {
 
         if (util <= kink) {
             return ((util * multiplierPerBlock) / BASE) + baseRatePerBlock;
-        } else {
-            uint256 normalRate = ((kink * multiplierPerBlock) / BASE) + baseRatePerBlock;
-            uint256 excessUtil;
-            unchecked {
-                excessUtil = util - kink;
-            }
-            return ((excessUtil * jumpMultiplierPerBlock) / BASE) + normalRate;
         }
+        uint256 normalRate = ((kink * multiplierPerBlock) / BASE) + baseRatePerBlock;
+        uint256 excessUtil;
+        unchecked {
+            excessUtil = util - kink;
+        }
+        return ((excessUtil * jumpMultiplierPerBlock) / BASE) + normalRate;
```

Comment:
We can cache `kink` variable in memory instead of perfoming multiple `SLOAD` operations

Reply:
Fixed in 1aaf8f8